tests/x509/test_x509.py

```diff
@@ -62,7 +62,10 @@ def __init__(self, value):
     value = utils.read_only_property("_value")
 
 
-def _load_cert(filename, loader, backend=None):
+T = typing.TypeVar("T")
+
+
+def _load_cert(filename, loader: typing.Callable[..., T], backend=None) -> T:
```

Comment:
Same deal, this shouldn't be `...`, this should just be `bytes`?

Reply:
Yes (although it'd be `[bytes]` I suppose). I skipped it for the same reason I skipped the last one: I cared exclusively about the return type and figured we could come back around and completely type it later.